server/lib/validation.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     INTEGER: joi.number().integer(),
     OFFSET: joi.number().integer().min(0),
     REFERRER: joi.string().max(2048).uri({ scheme: [ 'android-app', 'http', 'https' ]}).allow('none'),
+    RESUME: joi.string().regex(/^[\w]+={0,2}/), // match any alpha numberic string that could end with = or ==
```

Comment:
I am not 100% sure, but the resume token uses `atob` and `btoa` encoding/decoding functions. The [docs](https://developer.mozilla.org/en-US/docs/Web/API/WindowBase64/Base64_encoding_and_decoding) for these doesn't really specify anything except that it is base64. That considered, I think I should updated the regex to support `/` and `+` characters.

@shane-tomlinson Are we missing anything?